.github/workflows/windows_conditional_compilation.yml

```diff
@@ -133,10 +128,17 @@ jobs:
             -S ${{ env.OPENVINO_REPO }} `
             -B ${{ env.BUILD_DIR }}
 
+      - name: Clean sccache stats
+        run: '& "$Env:SCCACHE_PATH" --zero-stats'
+
       - name: Cmake build - CC COLLECT
-        run: |
-          cmake --build ${{ env.BUILD_DIR }} --parallel --config ${{ env.CMAKE_BUILD_TYPE }}
-          cmake --build ${{ env.BUILD_DIR }} --parallel --config ${{ env.CMAKE_BUILD_TYPE }} --target sea_itt_lib
```

Comment:
@akashchi how can we guarantee that multi line steps fail as expected? I want to avoid splitting into multiple steps

Reply:
The `pwsh` does not have something like `-e` that is set for the [`bash` in GHA](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell). It seems we could set the `ErrorActionPreference` variable to `Stop` in the environment - that should tell the executor to stop the process on error, the default value of `ErrorActionPreference` is `Continue`: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_preference_variables?view=powershell-7.4#erroractionpreference

Reply:
can we combine multiple commands via `&&` then?

Reply:
Yes: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_pipeline_chain_operators?view=powershell-7.4 if the powershell's version is > 7.4. I guess @ababushk could provide version info.

Reply:
We use exactly Powershell 7.4.0, which is the latest version. Let me try to combine the commands with `&&`, but the build is fine right now (although I can increase number of threads in order to make it fail again), so we won't see any effect